node/tests/data_request_examples.rs

```diff
@@ -417,4 +436,36 @@ mod examples {
             fee: 0,
         }
     }
+
+    pub fn random_bytes() -> BuildDrt {
+        let url_0 = "";
+        let r0_script = vec![];
```

Comment:
This results in an error because an empty script must be represented as an empty CBOR array:

```suggestion
        let r0_script = vec![128];
```

This test does not detect that error because it doesn't validate the request, I will try to fix that and open a PR.

Reply:
Opened #2062